pkgs/development/python-modules/plum_py/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitLab, isPy3k, pytest, baseline }:
+
+buildPythonPackage rec {
+  pname = "plum_py";
+  version = "0.4.0";
+  disabled = !isPy3k;
+
+  src = fetchFromGitLab {
+    owner = "dangass";
+    repo = "plum";
+    rev = "6a9ff863c0e9fa21f7b2230d25402155a5522e4b";
+    sha256 = "1iv62yb704c61b0dvsmyp3j6xpbmay532g9ny4pw4zbg3l69vd5j";
+  };
+
+  patchPhase = ''
```

Comment:
So, this package has some failing tests. The specific failures are just some changes in the exact value of `repr` on some enums. The code actually has a special case to fix up these repr values, but it's disabled for >= python 3.7 for some reason. This just enables it for all versions (it's harmless if it's not needed).

I filed this with upstream here: https://gitlab.com/dangass/plum/-/issues/126

Is this the sort of thing that's acceptable in nixpkgs, or do I have to wait for upstream to fix it?

Reply:
We can also ignore those tests with disabledTests.

Reply:
Assuming that the tests express the intent of the code, then fixing it is certainly better than just disabling the tests and leaving the code broken, right? (For all I know, it might even be a Nix-specific change to Python that's breaking them.)

I can understand not wanting this sort of thing in nixpkgs, and ideally it would be fixed upstream, but I haven't heard anything on the issue I filed yet.